portfolio/src/main/webapp/index.html

```diff
@@ -153,6 +169,14 @@ <h4>Recommend A Destination</h4>
                 placeholder="Description (What to do, where specifically to go, general comments, etc.)"
               ></textarea>
             </div>
+            <div class="form-group col-auto">
+              <label class="sr-only" for="displayedName">Displayed Name</label>
+              <input type="text" class="form-control" id="displayedName" name="displayedName" aria-describedby="displayedName" placeholder="Displayed Name" required>
```

Comment:
I'm a bit confused, aria-describedby has the id of its own element, is that intended?

Reply:
Oops, I took this from bootstrap and I forgot to delete that part.